Source/monster.cpp

```diff
@@ -189,10 +189,7 @@ void InitMonster(MonsterStruct &monster, Direction rd, int mtype, Point position
 	}
 
 	if (!gbIsMultiplayer) {

```

Comment:
To keep this consistent with other single statement conditionals (most notably the identical code in PlaceUniqueMonst around line 509) the braces should be removed here.

Reply:
ahh you are right. Updated the pull request. Thank you @ephphatha !